packages/typebox-validator/src/index.ts

```diff
@@ -72,6 +72,15 @@ export function tbValidator<
       }
       return data
     }
-    return c.json({ success: false, errors: [...Value.Errors(schema, data)] }, 400)
+    
+    const errors = Array.from(Value.Errors(schema, data));
```

Comment:
I think it's not necessary to make it as an array. How about using the return object from `Value.Errors()` directly?

```ts
const errors = Value.Errors(schema, data)
```